src/Instance/InstanceDetails/MemberTenantHoldings/MemberTenantHoldings.js

```diff
@@ -40,10 +40,10 @@ const MemberTenantHoldings = ({
   const { holdingsRecords, isLoading } = useInstanceHoldingsQuery(instance?.id, { tenantId: id });
   const isUserInCentralTenant = checkIfUserInCentralTenant(stripes);
 
-  const canViewHoldings = hasMemberTenantPermission('ui-inventory.instance.view', id, userTenantPermissions);
-  const canCreateItem = hasMemberTenantPermission('ui-inventory.item.edit', id, userTenantPermissions);
-  const canCreateHoldings = hasMemberTenantPermission('ui-inventory.holdings.edit', id, userTenantPermissions);
-  const canViewItems = hasMemberTenantPermission('ui-inventory.instance.view', id, userTenantPermissions);
+  const canViewHoldings = hasMemberTenantPermission('ui-inventory.holdings.create', id, userTenantPermissions);
```

Comment:
Please also update variable names

Reply:
Hi @BogdanDenis To view and create holdings we need "View, create holdings" permission (ui-inventory.holdings.create).  **canViewHoldings** is used for "View holdings" button. **canCreateHoldings** is used for "Add holdings" button

Reply:
It's confusing to me why there are two variables that are checking for the same permission and they are called differently.
Additionally, from variable name `canViewHoldings` I expect it to be true only when user has view permissions, but not edit or create.

Reply:
@BogdanDenis I was also confused about the fact that to view holdings you actually need to check for View and **create** permission, we discussed it with PO and figured out that there is no separate permission to just view holdings. I suggest renaming this variable to `canViewAndCreateHoldings` and using it in two places

Reply:
@OleksandrHladchenko1 please review comment from @mariia-aloshyna